lib/OrganizationsFieldArray/OrgInfo.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { isEmpty } from 'lodash';
+import { AppIcon } from '@folio/stripes/core';
+import { Field } from 'redux-form';
+import {
+  Button,
+  Col,
+  Card,
+  Layout,
+  Label,
+  Select,
+} from '@folio/stripes/components';
+import { required } from '../validators';
+
+export default class OrgInfo extends React.Component {
```

Comment:
Could we change this component to:
1. Not do the decision making of whether to render the org info vs org picker, moving that logic up to the `EditOrganizationCard`. So of its current responsibilities, remove everything that isn't rendering the org name and role.
1. Support rendering of interfaces if they are passed into it as a prop.
1. Rename this to `OrganizationCard`

?

If so, could we then use this component in _both_ the `EditOrganizationCard` as well as where we're currently rendering out the organizations in `ui-agreements/src/components/AgreementSections/Organizations.js`? That would be nice.

Reply:
In any case, the decision making that I say in bullet 1 should probably be moved up to that hypothetical `EditOrganizationCard` (what's currently just called `OrganizationCard`). This design results in a lot of props passing when it wouldn't be strictly necessary.